Character.cpp

```diff
@@ -86,32 +90,74 @@ int Character::takeDamage(int damage)
     return hitPoints;
 }
 
+int Character::getHP() const { return hitPoints; }
+int Character::getArmorLevel() const { return armor; }
+int Character::getAttackDamage() const { return attackDamage; }
+bool Character::getIsDefending() const { return isDefending; }
+
+const std::vector<std::unique_ptr<Item>>& Character::getHelpfulItems() const { return helpfulItems; }
+const std::vector<std::unique_ptr<Item>>& Character::getDefensiveItems() const { return defensiveItems; }
+
+void Character::boostArmor( int amount )
+{
+    armor += amount;
+    std::cout << getName() << "'s armor level has been boosted to " << armor << std::endl;
+}
+
+void Character::boostHitPoints( int amount )
+{
+    hitPoints += amount;
+    std::cout << getName() << "'s hit point level has been boosted to " << hitPoints << std::endl;
+}
+
+void Character::boostAttackDamage( int amount )
+{
+    attackDamage += amount;
+    std::cout << getName() << "'s attack damage level has been boosted to " << attackDamage << std::endl;
+}
```

Comment:
None of this code needed to be moved from the header file.
You must learn when existing code needs to be changed, and when it should be left alone.
- [ ] revert